Formula/stellar-cli.rb

```diff
@@ -0,0 +1,18 @@
+class StellarCli < Formula
+  desc "CLI for interacting with and deploying to Stellar"
+  homepage "https://developers.stellar.org/"
+  url "https://github.com/stellar/stellar-cli.git", tag: "v20.3.4"
```

Comment:
The stellar-cli wasn't released for v20.3.4, so I doubt this will work, but once v21 is a final release we'll want this exact file with v21.0.0 in it, so we could commit that ahead of time for ready.
```suggestion
  url "https://github.com/stellar/stellar-cli.git", tag: "v21.0.0"
```